x-pack/package.json

```diff
@@ -36,7 +36,6 @@
     "@mattapperson/slapshot": "1.4.0",
     "@storybook/addon-actions": "^5.2.6",
     "@storybook/addon-console": "^1.2.1",
-    "@storybook/addon-info": "^5.2.6",
     "@storybook/addon-knobs": "^5.2.6",
```

Comment:
do we still need all the storybook deps here? Why aren't those handled by `@kbn/storybook` instead?

Reply:
I believe we do, the reason is because I've setup the global Storybook for Canvas, too. It runs most of the Canvas stories but about 20 it cannot, either because of something stateful or because of imports from `ui/public`, thus, Canvas still has local Storybook to run those stories.